src/modules/formBuilder/components/FormEditorItemPreview.tsx

```diff
@@ -32,12 +32,12 @@ const FormEditorItemPreview: React.FC<FormEditorItemPreviewProps> = ({
         size='small'
         variant='gray'
       />
-      {toggleValue === 'readOnly' ? (
-        <DynamicViewField item={selectedItem} value={value} />
+      {toggleValue === 'readOnly' || item.readOnly ? (
```

Comment:
Looking at this in the context of the editing drawer, I think it would be a bit cleaner to hide the toggle, and just show the read-only view if "read only" is checked. I feel like that gets at the desired functionality, which is to preview what this field will look like when it's rendered within the form. (And the user would still be able to preview that read-only view by temporarily toggling it to read-only, if they wanted to.)

What do you think?
I also had some trouble distinguishing what was the preview vs what was items -- maybe some background color would help, but we can also run that question by design.

<img width="706" alt="Screenshot 2024-05-22 at 2 24 24 PM" src="https://github.com/greenriver/hmis-frontend/assets/5333982/8f62a732-7853-4f3b-8a3b-9ca07dfa6971">